src/main/java/nl/stokpop/lograter/processor/performancecenter/PerformanceCenterCalculator.java

```diff
@@ -61,37 +60,30 @@ public static List<Double> createDataSet(int count, double min, double max, doub
      * @return null when the aggregation period cannot be determined, otherwise the aggregation period in seconds
      */
     public static Integer determineAggregationPeriod(File resultsDatabaseFile) {
-        int aggregationPeriodInSeconds;
         try {
-             aggregationPeriodInSeconds = PerformanceCenterFileParser.fetchAnalysisAggregationPeriodInSeconds(resultsDatabaseFile.getAbsoluteFile().getParentFile());
-        } catch (IOException | LogRaterException e) {
+            return ResultConfigReader
```

Comment:
It seems that before 'null' was acceptable value in case of specific cases.
Here I actually changed the behaviour and made this field "required" so if the setting is missing or value cannot be parsed - now it fails.

This was done according to the request on using primitive values.
If this setting is actually nullable - I can revert to autoboxing